### ISSUE TITLE
Package in conda

### ISSUE THREAD

**Issue by cmutel** [pjamesjoyce/lcopt#9]
Recipe is here: https://github.com/cmutel/conda-recipes/blob/master/lcopt/meta.yaml

Create new build environment; you will need packages anaconda, conda, conda-build.

Can then run `conda build -c defaults -c cmutel -c conda-forge -c haasad lcopt` and upload to anaconda; installation instructions will then be as simple as:

`conda install -y -q -c conda-forge -c pjamesjoyce -c cmutel -c haasad lcopt`

**Comment by pjamesjoyce** [comment]
This was far more difficult than it should have been! I think it's done now though. Thanks for the recipe, that was a great help.

I had to increment the version number up to 0.4.2 to get it onto pypi (0.4.1 had an uncaught mistake in the entry points). I'm not sure if this affects the *Version* checkbox in the review (openjournals/joss-reviews#339) - perhaps @katyhuff or @amoeba could advise?

I've installed it from conda using the command above into a clean environment on my windows machine and it runs as expected.
